test/passing/break_cases-align.ml.ref

```diff
@@ -25,8 +25,7 @@ let f =
      |M ->
     match y with
     | O -> 5
-    | P when h x -> (
-        function
+    | P when h x -> ( function
```

Comment:
This kind of change is unavoidable with this new design

Reply:
Why ? This is not a collection expression.